webapp/src/components/react_select_setting.tsx

```diff
@@ -83,7 +83,11 @@ export default class ReactSelectSetting extends React.PureComponent {
                 {...this.props}
             >
                 <AsyncSelect
-                    {...this.props}
```

Comment:
This have a bit of "behavior change" because you are no longer able to pass whatever you want to the component and it blindly pass to react-select, I you have components passing things to react-select-setting to affect the underneath react-select, you need to pass it in the react-select-settings properties explicitly.

Reply:
Actually I saw that is something that is done a lot, passing things like isMulti or label.

Reply:
I think now all the values are passed correctly (I'm not passing ref or key because I guess was expected to be passed to the parent, no to the ReactSelect component)